Expensify/App
# [No QA]Fix iOS production deploys

### Details
Debugged fastlane locally and was able to submit production builds using this code. I believe production should fail once more, then it should run this code and succeed.

### Fixed Issues

$ https://github.com/Expensify/App/issues/4978

### Tests
I ran these locally, if you do not have the secrets decrypted this will not work
1. Set the version `export VERSION='1.0.92.0'
2. Run `bundle exec fastlane ios production`
3. Verify you see the build submitted for review 🎉
